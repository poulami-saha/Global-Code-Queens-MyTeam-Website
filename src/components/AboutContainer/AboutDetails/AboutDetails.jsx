import { Typography } from "../../utility";
import styles from "./AboutDetails.module.css";

const AboutDetails = () => {
  return (
    <section className={styles.container}>
      <Typography elType="h1Small">About</Typography>
      <article className={styles.aboutus}>
        <div className={styles.bar}></div>
        <Typography elType="body1">
          We help companies build dynamic teams made up of top global talent.
          Using our network of passionate professionals we drive innovation and
          deliver incredible outcomes. Talented, diverse teams shape the best
          products and experiences. Weâ€™ll bring those teams to you.
        </Typography>
      </article>
    </section>
  );
};

export default AboutDetails;
