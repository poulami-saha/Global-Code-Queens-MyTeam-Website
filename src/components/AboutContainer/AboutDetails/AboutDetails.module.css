.container {
  background: var(--midnight-green);
  color: var(--primary-white);
  text-align: center;
  padding-top: 20px;
  padding-bottom: 108px;
  background-image: url("../../../assets/bg-pattern-about-1-mobile-nav-1.svg");
  background-position: bottom -100px right -100px;
  background-repeat: no-repeat;
}

.body {
  display: flex;
  flex-direction: column;
  gap: 16px;
  align-items: center;
  justify-content: center;
  max-width: 100%;
}

@media (min-width: 768px) {
  .container {
    padding-top: 80px;
    padding-bottom: 112px;
    background-position-y: bottom;
  }

  .body {
    gap: 24px;
  }

  .aboutus {
    width: 59vw;
  }
}

@media (min-width: 960px) {
  .container {
    padding-top: 60px;
    padding-bottom: 112px;
  }

  .body {
    flex-direction: row;
    gap: 24px;
    justify-content: space-between;
  }

  .headline {
    align-self: flex-start;
    margin-top: 40px;
  }

  .bar {
    width: 50px;
    height: 4px;
    background: var(--coral-color);
    margin-block: 40px;
  }

  .aboutus {
    text-align: left;
    align-items: center;
    width: 50vw;
  }
}
