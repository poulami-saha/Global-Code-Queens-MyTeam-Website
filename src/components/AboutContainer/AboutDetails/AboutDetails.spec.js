import React from "react";
import { render } from "@testing-library/react";
import AboutDetails from "./AboutDetails";

test("renders NotFound correctly and matches snapshot", async () => {
  const { asFragment, getByText } = render(<AboutDetails />);

  // Snapshot test
  expect(asFragment()).toMatchSnapshot();

  // Content test
  expect(
    getByText(
      /We help companies build dynamic teams made up of top global talent. Using our network of passionate professionals we drive innovation and deliver incredible outcomes. Talented, diverse teams shape the best products and experiences. Weâ€™ll bring those teams to you./
    )
  ).toBeInTheDocument();
});
