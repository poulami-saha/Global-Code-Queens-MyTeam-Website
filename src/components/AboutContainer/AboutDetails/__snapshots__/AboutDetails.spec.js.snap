// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AboutDetails correctly and matches snapshot 1`] = `
<DocumentFragment>
  <section
    class="container"
  >
    <h1
      class="h1Small"
      id=""
    >
      About
    </h1>
    <article
      class="aboutus"
    >
      <div
        class="bar"
      />
      <p
        class="body1"
        id=""
      >
        We help companies build dynamic teams made up of top global talent. Using our network of passionate professionals we drive innovation and deliver incredible outcomes. Talented, diverse teams shape the best products and experiences. We’ll bring those teams to you.
      </p>
    </article>
  </section>
</DocumentFragment>
`;

exports[`renders AboutDetails correctly and matches snapshot 1`] = `
<DocumentFragment>
  <section
    class="container"
  >
    <h1
      class="h1Small"
      id=""
    >
      About
    </h1>
    <article
      class="aboutus"
    >
      <div
        class="bar"
      />
      <p
        class="body1"
        id=""
      >
        We help companies build dynamic teams made up of top global talent. Using our network of passionate professionals we drive innovation and deliver incredible outcomes. Talented, diverse teams shape the best products and experiences. We’ll bring those teams to you.
      </p>
    </article>
  </section>
</DocumentFragment>
`;
